kotlin/src/test/kotlin/org/jdbi/v3/core/kotlin/KotlinMapperTest.kt

```diff
@@ -40,6 +40,22 @@ class KotlinMapperTest {
 
     }
 
+    @Test fun shouldUseSecondOccurrenceOfColumnWhenMultipleColumnsWithSameNameArePresent() {
+
+        val mapper = KotlinMapper(DataClassWithOnlyPrimaryConstructor::class.java)
+        mockColumns("id", "name", "name")
```

Comment:
Say we want to search for occupations by name, matching on word boundaries. In Postgres you could use regex query like this: `l.name  ~* ('\\y' || :searchTerm)`. When we return occupations we want to highlight the search term within the  name using markdown syntax,  plus I want to return all other fields as in `SELECT l.*, regexp_replace(l.name, '(\\y' || :searchTerm || ')', '*\\1*', 'ig') as name` . In this case l.* will also contain `name` column, however I want to make sure that the second occurrence with the search term highlighted is the one that gets mapped.